FirebaseDatabase/Tests/Unit/FLevelDBStorageEngineTests.m

```diff
@@ -469,29 +469,6 @@ - (void)testHugeLeafNodeThenDeeperSet {
                         NODE((@{@"deep" : @"deep-value"})));
 }
 
-// Well this is awkward, but NSJSONSerialization fails to deserialize JSON with tiny/huge doubles
-// It is kind of bad we raise "invalid" data, but at least we don't crash *trollface*
-- (void)testExtremeDoublesAsServerCache {
-#ifdef TARGET_OS_IOS
-  if ([[NSProcessInfo processInfo] operatingSystemVersion].majorVersion >= 11) {
```

Comment:
`TARGET_OS_IOS` is defined on visionOS but the `majorVersion` check should be `>= 1` on visionOS, causing the test to continue rather than returning. `TARGET_OS_XR` is not specified in the following `#if` either.

Rather than adjusting these checks, I removed the test entirely since our minimum deployment target is iOS 11+ and the first version of visionOS contains the `NSJSONSerialization` double serialization fix.